app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java

```diff
@@ -173,5 +178,77 @@ public Attendee(User user, String review, int rating) {
             this.rating = rating;
         }
     }
+
+    /**
+     * @param  {string} token  (token of the requesting user )
+     * @param  {int} id (the id of the meeting)
+     * @param {int} rating (the rating of the meting)
+     * @param {boolean Array} goals (boolean array that states if the goals of the meeting have been achived or not)
+     * @param {JSONArray} attendees (JSOnArray that have the ratings and review of every member of the meeting)
+     * @param {HttpResponse} HTTP_RESPONSE (http interface instance which is the response coming from the server after logging in containing info of the user in the Database)
+     */
+    public void rate(String token,int id, int rating,  boolean [] goals  , JSONArray attendees,final HTTPResponse HTTP_RESPONSE){
```

Comment:
it is an instance method so you already have access to the id so remove it from the parameters. and edit the method documentation accordingly.